### PR TITLE
[zh-cn]: Update translation in 'svg/tutorial/getting_started'

### PR DESCRIPTION
**Pull request by WBbug**
### Description
Update translation  in svg/tutorial/getting_started

### Additional details
https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorial/Getting_Started#basic_properties_of_svg_files

### CODE REVIEW

File: files/zh-cn/web/svg/tutorial/getting_started/index.md [hunk 1]
```diff
@@ -26,66 +26,64 @@ slug: Web/SVG/Tutorial/Getting_Started
 </svg>
 ```
 
-复制并粘贴代码到文件 demo1.svg。然后用 Firefox 打开该文件。它将会呈现为下面的截图。(Firefox 用户点击[这里](https://mdn.dev/archives/media/attachments/2012/07/09/3075/89b1e0a26e8421e19f907e0522b188bd/svgdemo1.xml))
+复制并粘贴代码到文件 demo1.svg。然后在浏览器中打开该文件。它将会呈现为下面的截图。(Firefox 用户点击[这里](https://mdn.dev/archives/media/attachments/2012/07/09/3075/89b1e0a26e8421e19f907e0522b188bd/svgdemo1.xml))
```
Comment (jasonren0403) [inline files/zh-cn/web/svg/tutorial/getting_started/index.md:29]:
```suggestion
复制并粘贴代码到文件 demo1.svg。然后在浏览器中打开该文件。它将会呈现为下面的截图。（Firefox 用户点击[这里](https://mdn.dev/archives/media/attachments/2012/07/09/3075/89b1e0a26e8421e19f907e0522b188bd/svgdemo1.xml)）
```

File: files/zh-cn/web/svg/tutorial/getting_started/index.md [hunk 2]
```diff
@@ -26,66 +26,64 @@ slug: Web/SVG/Tutorial/Getting_Started
...
 
 绘制流程包括以下几步：
 
-1. 从`svg`根元素开始：
+1. 从 {{SVGElement("svg")}} 根元素开始：
 
     - 应舍弃来自 (X)HTML 的 doctype 声明，因为基于 SVG 的 DTD 验证导致的问题比它能解决的问题更多。
-    - SVG 2 之前，version 属性和 baseProfile 属性用来供其他类型的验证识别 SVG 的版本。SVG 2 不推荐使用 version 和 baseProfile 这两个属性。
+    - SVG 2 之前 `version` 属性和 `baseProfile` 属性用来供其他类型的验证识别 SVG 的版本。SVG 2 不推荐使用 `version` 和 `baseProfile` 这两个属性。
     - 作为 XML 的一种方言，SVG 必须正确的绑定命名空间（在 xmlns 属性中绑定）。请阅读[命名空间速成](/zh-CN/docs/Web/SVG/Namespaces_Crash_Course) 页面获取更多信息。
```
Comment (jasonren0403) [inline files/zh-cn/web/svg/tutorial/getting_started/index.md:39]:
```suggestion
    - 作为 XML 的一种方言，SVG 必须正确的绑定命名空间（在 xmlns 属性中绑定）。请阅读[命名空间速成](/zh-CN/docs/Web/SVG/Namespaces_Crash_Course)页面获取更多信息。
```

File: files/zh-cn/web/svg/tutorial/getting_started/index.md [hunk 3]
```diff
@@ -26,66 +26,64 @@ slug: Web/SVG/Tutorial/Getting_Started
...
-            <iframe src="image.svg"></iframe>
+     ```html
+    <iframe src="image.svg"></iframe>
     ```
 
-  - 理论上同样可以使用 `img` 元素，但是在低于 4.0 版本的 Firefox 中不起作用。
-  - 最后 SVG 可以通过 JavaScript 动态创建并注入到 HTML DOM 中。这样具有一个优点，可以对浏览器使用替代技术，在不能解析 SVG 的情况下，可以替换创建的内容。
-
-  阅读[this dedicated article](/zh-CN/SVG_In_HTML_Introduction) 以深入了解该话题。
+  - 最后 SVG 可以通过 JavaScript 动态创建并注入到 HTML DOM 中。
```
Comment (jasonren0403) [inline files/zh-cn/web/svg/tutorial/getting_started/index.md:65]:
```suggestion
  - 最后，SVG 可以通过 JavaScript 动态创建并注入到 HTML DOM 中。
```

File: files/zh-cn/web/svg/tutorial/getting_started/index.md
Comment (jasonren0403) [inline files/zh-cn/web/svg/tutorial/getting_started/index.md:1]: 92-94行与原文不同步，请参照原文进行更新

File: files/zh-cn/web/svg/tutorial/getting_started/index.md [hunk 4]
```diff
@@ -26,66 +26,64 @@ slug: Web/SVG/Tutorial/Getting_Started
...
+复制并粘贴代码到文件 demo1.svg。然后在浏览器中打开该文件。它将会呈现为下面的截图。(Firefox 用户点击[这里](https://mdn.dev/archives/media/attachments/2012/07/09/3075/89b1e0a26e8421e19f907e0522b188bd/svgdemo1.xml))
 
 ![svgdemo1.png](svgdemo1.png)
 
 绘制流程包括以下几步：
 
-1. 从`svg`根元素开始：
+1. 从 {{SVGElement("svg")}} 根元素开始：
 
     - 应舍弃来自 (X)HTML 的 doctype 声明，因为基于 SVG 的 DTD 验证导致的问题比它能解决的问题更多。
```
Comment (yin1999) [inline files/zh-cn/web/svg/tutorial/getting_started/index.md:37]:
```suggestion
    - 应舍弃来自 (X)HTML 的 doctype 声明，因为基于 DTD 的 SVG 验证导致的问题比它能解决的问题更多。
```

**Review by jasonren0403 — DISMISSED** [review]
主要是标点符号问题

File: files/zh-cn/web/svg/tutorial/getting_started/index.md [hunk 5]
```diff
@@ -26,66 +26,64 @@ slug: Web/SVG/Tutorial/Getting_Started
...
-    - SVG 2 之前，version 属性和 baseProfile 属性用来供其他类型的验证识别 SVG 的版本。SVG 2 不推荐使用 version 和 baseProfile 这两个属性。
+    - SVG 2 之前 `version` 属性和 `baseProfile` 属性用来供其他类型的验证识别 SVG 的版本。SVG 2 不推荐使用 `version` 和 `baseProfile` 这两个属性。
```
Comment (yin1999) [inline files/zh-cn/web/svg/tutorial/getting_started/index.md:38]:
```suggestion
    - SVG 2 之前 `version` 属性和 `baseProfile` 属性用来供其他类型的验证识别 SVG 的版本。SVG 2 已弃用 `version` 和 `baseProfile` 这两个属性。
```

File: files/zh-cn/web/svg/tutorial/getting_started/index.md (hunk 2 above)
Comment (yin1999) [inline files/zh-cn/web/svg/tutorial/getting_started/index.md:39]:
```suggestion
    - 作为 XML 的一种方言，SVG 必须正确的绑定命名空间（在 xmlns 属性中绑定）。请阅读[命名空间速成](/zh-CN/docs/Web/SVG/Namespaces_Crash_Course)页面获取更多信息。
```

File: files/zh-cn/web/svg/tutorial/getting_started/index.md [hunk 6]
```diff
@@ -26,66 +26,64 @@ slug: Web/SVG/Tutorial/Getting_Started
...
     - 作为 XML 的一种方言，SVG 必须正确的绑定命名空间（在 xmlns 属性中绑定）。请阅读[命名空间速成](/zh-CN/docs/Web/SVG/Namespaces_Crash_Course) 页面获取更多信息。
 
-2. 绘制一个完全覆盖图像区域的矩形 `<rect/>，`把背景颜色设为红色。
-3. `一个半径 80px 的绿色圆圈<circle/>`绘制在红色矩形的正中央（向右偏移 150px，向下偏移 100px）。
+2. 绘制一个完全覆盖图像区域的矩形 {{SVGElement("rect")}} ，把背景颜色设为红色。
```
Comment (yin1999) [inline files/zh-cn/web/svg/tutorial/getting_started/index.md:41]:
```suggestion
2. 绘制一个完全覆盖图像区域的矩形 {{SVGElement("rect")}}，把背景颜色设为红色。
```

File: files/zh-cn/web/svg/tutorial/getting_started/index.md [hunk 7]
```diff
@@ -26,66 +26,64 @@ slug: Web/SVG/Tutorial/Getting_Started
...
-            <object data="image.svg" type="image/svg+xml" />
+    ```html
+    <object data="image.svg" type="image/svg+xml"></object>
     ```
 
   - 类似的也可以使用 `iframe` 元素引用 SVG 文件：
 
-    ```plain
-            <iframe src="image.svg"></iframe>
+     ```html
```
Comment (yin1999) [inline files/zh-cn/web/svg/tutorial/getting_started/index.md:61]:
````suggestion
    ```html
````

File: files/zh-cn/web/svg/tutorial/getting_started/index.md [hunk 8]
```diff
@@ -26,66 +26,64 @@ slug: Web/SVG/Tutorial/Getting_Started
...
 
-### SVG 文件类型
+## SVG 文件类型
 
 SVG 文件有两种形式。普通 SVG 文件是包含 SVG 标记的简单文本文件。推荐使用“.svg”（全部小写）作为此类文件的扩展名。
 
 由于在某些应用（比如地图应用等）中使用时，SVG 文件可能会很大，SVG 标准同样允许 gzip 压缩的 SVG 文件。推荐使用“.svgz”（全部小写）作为此类文件扩展名。不幸的是，如果服务器是微软的 IIS 服务器，使 gzip 压缩的 SVG 文件在所有的可用 SVG 的用户代理上可靠地起作用是相当困难的，而且 Firefox 不能在本地机器上加载 gzip 压缩的 SVG 文件。除非知道处理发布内容的 Web 服务器可以正确的处理 gzip，否则要避免使用 gzip 压缩的 SVG。
 
-### 关于 Web 服务器的小提示
+## 关于 Web 服务器上 .svgz 文件的小提示
```
Comment (yin1999) [inline files/zh-cn/web/svg/tutorial/getting_started/index.md:75]:
```suggestion
## 关于 web 服务器上 .svgz 文件的小提示
```

**Review by yin1999 — COMMENTED** [review]
Some suggestions

File: files/zh-cn/web/svg/tutorial/getting_started/index.md [hunk 9]
```diff
@@ -26,72 +26,70 @@ slug: Web/SVG/Tutorial/Getting_Started
...
 
-```plain
+```http
 Content-Type: image/svg+xml
 Content-Encoding: gzip
 Vary: Accept-Encoding
 ```
 
-可以利用[Network Monitor panel](/zh-CN/docs/Tools/Network_Monitor#Headers)或者[web-sniffer.net](http://web-sniffer.net/)之类的网站来检查服务器是否给 SVG 文件发送正确的 HTTP 头，向[web-sniffer.net](http://web-sniffer.net/)提交你的一个 SVG 文件的链接，然后查看 HTTP 响应头。如果发现服务器没有发送上述的响应头部值，那么你应该联系你的服务器供应商。如果不能说服他们为 SVG 修正服务器配置，可能还有一些我们可以自行解决的办法。请阅读 SVG 维基的[server configuration page](http://svg-whiz.com/wiki/index.php?title=Server_Configuration)以找到一些简单的解决方案。
+你可以利用[Network Monitor panel](https://firefox-source-docs.mozilla.org/devtools-user/network_monitor/index.html#headers)或者[websniffer.cc](https://websniffer.cc/)之类的网站来检查服务器是否给 SVG 文件发送正确的 HTTP 头。提交你的一个 SVG 文件的 URL，然后查看 HTTP 响应头。如果发现服务器没有发送上述的响应头部值，那么你应该联系你的服务器供应商。如果不能说服他们为 SVG 修正服务器配置，可能还有一些我们可以自行解决的办法。请阅读 w3.org 上的[server configuration page](https://www.w3.org/services/svg-server/)以找到一些简单的解决方案。
```
Comment (yin1999) [inline files/zh-cn/web/svg/tutorial/getting_started/index.md:92]: see:

- https://pontoon.mozilla.org/zh-CN/firefox/devtools/client/startup.properties/?string=155106
- https://github.com/mdn/translated-content/pull/12858/files#diff-e9a8d61dffc0528bfefd1a12a8d0860904f561a9ac172fcc35d2b58e5a5392a7R17

```suggestion
你可以利用[网络监控面板](https://firefox-source-docs.mozilla.org/devtools-user/network_monitor/index.html#headers)或者 [websniffer.cc](https://websniffer.cc/) 之类的网站来检查服务器是否给 SVG 文件发送正确的 HTTP 标头。提交你的一个 SVG 文件的 URL，然后查看 HTTP 响应标头。如果发现服务器没有发送具有上述值的响应标头，那么你应该联系你的网站托管服务商。如果不能说服他们为 SVG 修正服务器配置，可能还有一些我们可以自行解决的办法。请阅读 w3.org 上的[服务器配置页面](https://www.w3.org/services/svg-server/)以找到一些简单的解决方案。
```

File: files/zh-cn/web/svg/tutorial/getting_started/index.md [hunk 10]
```diff
@@ -26,72 +26,70 @@ slug: Web/SVG/Tutorial/Getting_Started
...
 
 服务器配置错误是 SVG 加载失败的常见原因，所以一定要确保你的服务器配置正确。如果不能把服务器配置成给 SVG 文件发送正确的响应头，这时 Firefox 很有可能把该文件的标记显示成文本或乱码，甚至会要求查看者选择打开文件的应用程序。
```
Comment (yin1999) [inline files/zh-cn/web/svg/tutorial/getting_started/index.md:94]:
```suggestion
服务器配置错误是 SVG 加载失败的常见原因，所以一定要确保你的服务器配置正确。如果不能把服务器配置成给 SVG 文件发送正确的响应标头，这时 Firefox 很有可能把该文件的标记显示成文本或乱码，甚至会要求查看者选择打开文件的应用程序。
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**